### ISSUE TITLE
As a user (volunteer and admin), I want creating events to be easy

### ISSUE THREAD

**Issue by samzx** [zendesk/volunteer_portal#156]
Current forms are clunky and time intensive to use. Users dread making events because of the bad user experience with the forms.

There should be a new version of forms which is **intuitive, quick and simple**

**Comment by samzx** [comment]
Possibly look at using Zendesk Garden components to help in this effort.

**Comment by samzx** [comment]
depends on #182
